INN/umbrella-boilerplate
# Clean up docs for GitHub template actions

## Changes

This pull request makes the following changes:

- merges `docs/README.md` and `readme-template.md` into `README.md`
- adds a link that creates a repo-setup issue in the template-generated repo
- improves and simplifies instructions now that we're no longer migrating from per-child-theme submodules on one giant multisite umbrella

## Why



For #4

## Testing/Questions

Features that this PR affects:

- docs
- `initialize.sh`


Questions that need to be answered before merging:

- [x] Are there loopholes in the docs?

Steps to test this PR:

1. Clone this repo someplace, then run `initialize.sh` and fill out `README.md`
2. Click the link in `README.md` to make sure that it fills out a new issue with an appropriate checklist.
